sdk/dapp-kit/src/hooks/useResolveSuiNSNames.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useSuiClientQuery } from './useSuiClientQuery.js';
+
+export function useResolveSuiNSName(address?: string | null) {
+	const { data, ...rest } = useSuiClientQuery(
+		'resolveNameServiceNames',
+		{
+			address: address!,
+			limit: 1,
+		},
+		{
+			enabled: !!address,
+			refetchOnWindowFocus: false,
```

Comment:
Should we define all of the refetch config instead of just this? e.g. `refetchInterval`